spec/views/idv/how_to_verify/show.html.erb_spec.rb

```diff
@@ -18,6 +18,13 @@
     end
 
     context 'renders the show template with' do
+      it('a step indicator with Getting started as the current step') do
```

Comment:
Conventionally, we don't include parentheses for RSpec BDD methods.

I would also personally expect the "it" parameterization to read as a human-readable sentence. Currently it lacks a verb of what "it" is doing.

```suggestion
      it 'renders a step indicator with Getting started as the current step' do
```

Reply:
@aduth   Ahh, good eye!  Thank you 
Updates made [here](https://github.com/18F/identity-idp/pull/11314)

Reply:
Updates merged in. Thanks for the review